.circleci/config.yml

```diff
@@ -66,15 +61,11 @@ jobs:
               yarn test:unit -- --mocha.bail false
               ./cc-test-reporter after-build --coverage-input-type lcov --exit-code $?
             fi
-
       - run:
           name: Rendering tests
-          command: |
-            DocId=$(curl -s --data-binary @data/apps/Executive_Dashboard.qvf http://localhost:9076/v1/apps/import | jq -r '.attributes.id')
-            APP_ID=$DocId yarn run test:rendering
-
+          command: yarn test:regression
       - store_artifacts:
-          path: ./test/rendering/artifacts
+          path: ./test/regression/artifacts
```

Comment:
These tests are still rendering tests, right? So I would prefer we keep having them under `test/rendering`